apps/dashboard/web/polaris_web/web/src/util/func.js

```diff
@@ -229,13 +232,11 @@ prettifyEpoch(epoch) {
 
       case "STOPPED":
         return{
-          color: "critical",
           tooltipContent: "Error occurred while running the test.",
```

Comment:
is this color field not being used anywhere ?

Reply:
added check